src/Lykke.Service.LoggingAdapter.Services/HealthNotification/HealthNotificationWriter.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Linq;
+using Lykke.Common.Log;
+using Lykke.Service.LoggingAdapter.Core.Domain.HealthNotification;
+using Lykke.Service.LoggingAdapter.Core.Domain.Log;
+
+namespace Lykke.Service.LoggingAdapter.Services.HealthNotification
+{
+    public class HealthNotificationWriter : IHealthNotificationWriter
+    {
+        public void SendNotification(IHealthNotifier healthNotifier, LogInformationDto logInformation)
+        {
+            healthNotifier.Notify(PrepareHealthMessage(logInformation), logInformation.Context);
+        }
+
+        private string PrepareHealthMessage(LogInformationDto source)
+        {
+            var propsDesctiptions = new []
```

Comment:
Not sure, if we need all this stuff in monitoring. Message will be enough. In the future we could pass all of this daa to the underlying log (Lykke.Logs should be updated to support this).